composer.json

```diff
@@ -20,13 +20,14 @@
     "require": {
         "php":                  "^7.4.0 || ^8.0",
         "doctrine/orm":         "^2.9.5",
-        "doctrine/persistence": "^2.2.2"
+        "doctrine/persistence": "^2.2.2 || ^3.0.2"
     },
     "require-dev": {
         "doctrine/coding-standard": "^9.0.0",
         "roave/infection-static-analysis-plugin": "^1.10.0",
         "phpunit/phpunit": "^9.5.9",
-        "vimeo/psalm": "^4.10.0"
+        "vimeo/psalm": "^4.10.0",
+        "doctrine/annotations" : "^1.13.2"
```

Comment:
Is this required? We should conflict with `doctrine/annotations < 1.13.2` perhaps?

Reply:
it's required by this -> examples/bootstrap-orm.php

Reply:
Alrighty :+1: